### DISCUSSION TITLE
Why there are 1.23 release note?

### DISCUSSION THREAD

**Discussion by pedrozimmermannsoares** [tprouvot/Salesforce-Inspector-reloaded#400] [General]
Exists 1.23 release note, but doesn't exist this version on github.

https://tprouvot.github.io/Salesforce-Inspector-reloaded/release-note/#version-123

**Comment by tprouvot** [comment]
Hi @rotivordep,
Not sure to understand your comment, do you mean that the documentation website includes the 1.23 release note, but the main version of reloaded is in version 1.22 ? 

The beta version is on the 1.23 that's why it is also available on the release notes.

**Reply by pedrozimmermannsoares** [reply]
Tks. Understood!
